index.js

```diff
@@ -40,6 +42,9 @@ function Consumer(options) {
   this.messageAttributeNames = options.messageAttributeNames || [];
   this.stopped = true;
   this.batchSize = options.batchSize || 1;
+  this.visibilityTimeout = options.visibilityTimeout || 0;
```

Comment:
Just talking about changing this from

```
options.visibilityTimeout || 0;
```

to

```
options.visibilityTimeout;
```

This way we don't need to care what the default value (`0`) is.